sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -307,7 +378,7 @@ private static bool IsFinalState(Response response, HeaderSource headerSource, o
 
             if (response.Status is >= 200 and <= 204)
             {
-                if (response.ContentStream is {Length: > 0})
+                if (response.ContentStream is { Length: > 0 })
```

Comment:
We should be using `Content` not `ContentStream` on Response.

Reply:
This only changes the format here, if you think this needs to be addressed, I would rather have another PR for this change.
Not quite sure about the logic here, it reads `ContentStream` and move the position back to 0.